ansible/host_vars/node-01.do-ams3.nimbus.fluffy.yml

```diff
@@ -0,0 +1,18 @@
+---
+# Necessary in order to build fluffy on the droplet with 1GB RAM
+swap_file_size_mb: 6144
+
+nimbus_fluffy_service_name: 'nimbus-fluffy-{{ nimbus_fluffy_network }}-{{ nimbus_fluffy_repo_branch }}-01'
+nimbus_fluffy_listening_addr: '167.99.215.25'
```

Comment:
Seems unnecessary if it's just the only node on the host. The default of `'0.0.0.0'` is enough.